ext/intl/tests/transliterator_child.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Subclass Transliterator
+--EXTENSIONS--
+intl
+--FILE--
+<?php
+class ChildTransliterator extends Transliterator
+{
+    public readonly string $id;
+
+    public static function new()
+    {
+        $tr = (new \ReflectionClass(ChildTransliterator::class))->newInstanceWithoutConstructor();
+        $tr->id = 'abc';
+
+        return $tr;
+    }
+}
+
+
+$tr = Tr::new();
+
+echo $tr->id;
```

Comment:
I'd prefer to use `var_dump()` here:
```suggestion
var_dump($tr->id);
```
That would require to adapt the test expection, too.

Reply:
updated thanks